### ISSUE TITLE
after updating to the latest version, the input is not minified anymore

### ISSUE THREAD

**Issue by stryju** [tleunen/brfs-htmlmin#3]
this is mot likely due to updated version of html-minifier introduced in https://github.com/stryju/brfs-htmlmin/commit/e52be79e59933174f141b203a6e659ad4b927660

culprit: https://github.com/kangax/html-minifier/compare/v0.6.9...v0.7.2

as per https://github.com/tleunen/brfs-htmlmin/pull/2#issuecomment-122903923

**Comment by tleunen** [comment]
I just tested it with the example from the repo and everything works as expected.

Could you provide an example of files that is not minified when using `brfs-htmlmin`?

**Comment by stryju** [comment]
ok, so it turns out i expected options to be merged, not override - that's why i was getting different result...

all good, thanks for the time you took to check that!

**Comment by tleunen** [comment]
Yep. I also think it would be better to merge the options instead of overriding them.

**Comment by stryju** [comment]
you can do that quite easily, tbh:

``` js
var extend = require( 'util' )._extend;

extend( target, config1, config2 );
```

but that would be a minor, if not major (breaking) change

**Comment by tleunen** [comment]
done and published as 2.0
